Makefile

```diff
@@ -1,8 +1,12 @@
 ORG     ?= $(shell basename $(realpath ..))
 PKGS    := $(shell go list ./... | grep -v /vendor/)
 
-build:
-	go build ${TARGETS}
+dep: ## Get build dependencies
+	go get -v -u github.com/golang/dep/cmd/dep
```

Comment:
Great!
Excuse me but I would tweak this a bit when I enable build cache in CircleCI so that you don't need to re-download the whole depencies on each build.

Perhaps I'll make build depends on `dep`, dep runs `go mod download`.